app/ui/client/lib/UserCard.js

```diff
@@ -0,0 +1,38 @@
+import { Tracker } from 'meteor/tracker';
+
+import { createEphemeralPortal } from '../../../../client/reactAdapters';
+
+const Dep = new Tracker.Dependency();
+let state;
+let dom;
+let unregister;
+const createAchor = () => {
+	const div = document.createElement('div');
+	div.id = 'react-user-card';
+	document.body.appendChild(div);
+	return div;
+};
+
+
+export const close = () => {
+	if (!dom) {
+		return;
+	}
+	unregister();
+	unregister = undefined;
+};
+
+const props = () => {
+	Dep.depend();
+	return state;
+};
+
+export const open = async ({ ...args }) => {
+	dom = dom || createAchor();
+	state = {
+		onClose: close,
+		...args,
+	};
+	Dep.changed();
+	unregister = unregister || await createEphemeralPortal(() => import('../../../../client/channel/UserCard'), props, dom);
+};
```

Comment:
Is this the suggested way to use this API?

```
import * as UserCard from './app/ui/client/lib/UserCard';
```

I remember @sampaiodiego saying something about using the upmost module to be used as an interface, therefore:

```
// 1st option
import { UserCard } from './app/ui/client';
// 2nd option
import { openUserCard, closeUserCard } from './app/ui/client';
```

This can't be done with named exports here.